logging.endpoint.http/src/swagger_controller.ts

```diff
@@ -1,8 +1,10 @@
-import {HttpRequestWithIdentity} from '@atlas-engine/http_contracts';
-
 import {Response} from 'express';
 import * as path from 'path';
+import {injectable} from 'inversify';
```

Comment:
Bitte alphabetische Sortierung aufrecht erhalten

```suggestion
import {Response} from 'express';
import {injectable} from 'inversify';
import * as path from 'path';
```